js/templates/modals/editListing/editListing.html

```diff
@@ -33,26 +33,19 @@ <h1 class="h3 txUnb padMd js-listingHeading"><% print(ob.createMode ? ob.polyT('
                 <input type="text" class="clrBr clrP" name="listing.item.title" id="editListingTitle" value="<%= ob.listing.item.title %>">
               </div>
             </div>
-            <div class="flexRow">
-              <div class="col12">
-                <label for="editListingSlug" class="required"><%= ob.polyT('editListing.slug') %></label>
-                <% if (ob.errors['listing.slug']) print(ob.formErrorTmpl({ errors: ob.errors['listing.slug'] })) %>
-                <input type="text" class="clrBr clrP" name="listing.slug" id="editListingSlug" value="<%= ob.listing.slug %>" placeholder="Enter a url fragment representing the listing, for example my-big-brown-shoe">
-              </div>
-            </div>
             <div class="flexRow gutterH">
-              <div class="col6">
-                <label for="editformat" class="required"><%= ob.polyT('editListing.type') %></label>
+              <div class="col6 simpleFlexCol">
+                <label for="editContractType" class="required"><%= ob.polyT('editListing.type') %></label>
                 <% if (ob.errors['listing.metadata.contractType']) print(ob.formErrorTmpl({ errors: ob.errors['listing.metadata.contractType'] })) %>
-                <select id="editformat" name="listing.metadata.contractType" class="clrBr clrP">
+                <select id="editContractType" name="listing.metadata.contractType" class="clrBr clrP marginTopAuto">
                   <% ob.contractTypes.forEach((contractType) => { %>
                   <option value="<%= contractType.code %>" <% if (contractType.code === ob.listing.metadata.contractType) print('selected') %>><%= contractType.name %></option>
                   <% }); %>
                 </select>
               </div>
-              <div class="col6">
+              <div class="col6 simpleFlexCol">
```

Comment:
I'm using this class to help with bottom aligning fields so they look nicer when there's errors on fields that are in columns. I tried using the `flexCol` and `flexKidsExpand` classes for this, but they were causing the Select2 fields to not be 100% wide.

Anyhow, if it's important enough to you and you want to play around with it, feel free to submit a PR.